### ISSUE TITLE
GPL License Violation - Please release source code or you will be reported for hosting malware

### ISSUE THREAD

**Issue by cryptochasers** [Enemy-Mine/ccminer#4]
Releasing a binary executable without releasing the source code is in clear violation of the GPL license, due to suspected malware you may also be reported. We highly recommend you post the corresponding source code to your latest release of the ccminer-enemy executable such that users can run and compile it as well as do security audits. 

The GNU General Public License states:
1. Source Code.
The “source code” for a work means the preferred form of the work for making modifications to it. “Object code” means any non-source form of a work.

A “Standard Interface” means an interface that either is an official standard defined by a recognized standards body, or, in the case of interfaces specified for a particular programming language, one that is widely used among developers working in that language.

The “System Libraries” of an executable work include anything, other than the work as a whole, that (a) is included in the normal form of packaging a Major Component, but which is not part of that Major Component, and (b) serves only to enable use of the work with that Major Component, or to implement a Standard Interface for which an implementation is available to the public in source code form. A “Major Component”, in this context, means a major essential component (kernel, window system, and so on) of the specific operating system (if any) on which the executable work runs, or a compiler used to produce the work, or an object code interpreter used to run it.

The “Corresponding Source” for a work in object code form means all the source code needed to generate, install, and (for an executable work) run the object code and to modify the work, including scripts to control those activities. However, it does not include the work's System Libraries, or general-purpose tools or generally available free programs which are used unmodified in performing those activities but which are not part of the work. For example, Corresponding Source includes interface definition files associated with source files for the work, and the source code for shared libraries and dynamically linked subprograms that the work is specifically designed to require, such as by intimate data communication or control flow between those subprograms and other parts of the work.

The Corresponding Source need not include anything that users can regenerate automatically from other parts of the Corresponding Source.

The Corresponding Source for a work in source code form is that same work.

**Comment by UselessGuru** [comment]
Source is available here:
https://github.com/Enemy-Mine/ccminer/releases/tag/v1.03-RVN

Just difficult to find...

**Comment by pallas1** [comment]
no, those are the sources of an old ccminer, just look at the file dates.
also, most algos are missing.

**Comment by UselessGuru** [comment]
> no, those are the sources of an old ccminer, just look at the file dates.
also, most algos are missing.

Just noted... publishing some old code is very bad camouflage. Very dodgy!!!

**Comment by nemosminer** [comment]
soucre code win: https://github.com/Enemy-Mine/ccminer/archive/v1.03-RVN.zip

source code linux : https://github.com/Enemy-Mine/ccminer/archive/v1.03-RVN.tar.gz

still Original code needs to be update the source code included with the release looks like new code?

**Comment by nemosminer** [comment]
very wrong this should not be allowed on GitHub

**Comment by ruplikminer** [comment]
Any news on this? So it's a MALWARE???

**Comment by kodeballer** [comment]
You could always sand box it and try and watch where else it connects to or gets updates from, or the author could just publish the source code
